SECURITY_INSIGHTS.yml

```diff
@@ -0,0 +1,104 @@
+header:
+  schema-version: 2.0.0
+  last-updated: '2025-01-21'
+  last-reviewed: '2025-01-21'
+  url: https://github.com/gittuf/gittuf
+
+project:
+  name: gittuf
+  administrators:
+    - name: Justin Cappos
+      affiliation: New York University
+      email: jcappos@nyu.edu
+      social: https://github.com/JustinCappos
+      primary: false
+    - name: Aditya Sirish A Yelgundhalli
+      affiliation: New York University
+      email: aditya.sirish@nyu.edu
+      social: https://github.com/adityasaky
+      primary: true
+    - name: Billy Lynch
+      affiliation: Chainguard
+      email: billy@chainguard.dev
+      social: https://github.com/wlynch
+      primary: false
+    - name: Reza Curtmola
+      affiliation: New Jersey Institute of Technology
+      email: reza.curtmola@njit.edu
+      social: https://github.com/reza-curtmola
+      primary: false
+    - name: Michael Lieberman
+      affiliation: Kusari
+      email: mike@kusari.dev
+      social: https://github.com/mlieberman85
+      primary: false
+  repositories:
```

Comment:
I haven't read the spec yet but if multiple repos are supported, should this go in /community or so?